pkg/services/mssqldr/database_pair_provision_for_existing_primary.go

```diff
@@ -0,0 +1,85 @@
+package mssqldr
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+	uuid "github.com/satori/go.uuid"
+)
+
+func (d *databasePairManagerForExistingPrimary) GetProvisioner(
+	service.Plan,
+) (service.Provisioner, error) {
+	return service.NewProvisioner(
+		service.NewProvisioningStep("preProvision", d.preProvision),
+		service.NewProvisioningStep(
+			"checkNameAvailability",
+			d.checkNameAvailability,
+		),
+		service.NewProvisioningStep("validatePriDatabase", d.validatePriDatabase),
+		service.NewProvisioningStep(
+			"deployPriARMTemplateForExistingInstance",
+			d.deployPriARMTemplateForExistingInstance,
+		),
+		service.NewProvisioningStep(
+			"deployFailoverGroupARMTemplateForExistingInstance",
+			d.deployFailoverGroupARMTemplateForExistingInstance,
+		),
+		service.NewProvisioningStep(
+			"deploySecARMTemplateForExistingInstance",
+			d.deploySecARMTemplateForExistingInstance,
```

Comment:
According to the doc, when using `azure-sql-12-0-dr-database-pair-from-existing-primary`, the user only has one primary database, and the secondary does not exist yet. Why use `deploySecARMTemplateForExistingInstance` here?

Reply:
The `azure-sql-12-0-dr-database-pair` uses it, too. Becauase https://github.com/Azure/open-service-broker-azure/blob/master/pkg/services/mssqldr/database_pair_provision.go#L29-L30. I'll also add the comment here.

Reply:
It makes sense. Thanks for the explanation.